### PR TITLE
Reorder entries in Misc/ACKS

### PR DESCRIPTION
**Pull request by merwok**

### CODE REVIEW

File: Misc/ACKS [hunk 1]
```diff
@@ -11,7 +11,9 @@ Without you, I would've stopped working on Python long ago!
 PS: In the standard Python distribution, this file is encoded in UTF-8
 and the list is in rough alphabetical order by last names.
 
+
```
Comment (merwok) [inline Misc/ACKS]:
```suggestion
```

File: Misc/ACKS [hunk 2]
```diff
@@ -1889,16 +1901,3 @@ Jelle Zijlstra
...
-Edison Abahurire
-Geoff Shannon
-Batuhan Taskaya
-Aleksandr Balezin
-Robert Leenders
-Tim Hopper
-Dan Lidral-Porter
-Ngalim Siregar
-Tim Gates
-Karthikeyan Singaravelan
```
Comment (merwok) [inline Misc/ACKS:1904]: 12 lines added, 13 lines removed because Ngalim Siregar was already in the right spot.

**Review by serhiy-storchaka — APPROVED** [review]
Thank you! I was going to do this myself, it was in my low priority list.

**Comment by merwok** [comment]
Thanks for review.  If the problem comes again, we could add a comment at the end of the file.

**Comment by serhiy-storchaka** [comment]
Nice idea about a comment at the end of the file!

**Comment by tirkarthi** [comment]
Sorry, being the latest entry added by myself I read the sorted nature of the file at the top but didn't that it was sorted by last name expecting sorted by first name. It's a good idea to add the comment in the end about the same. I tried adding a check to `patchcheck.py` since it already has a check for ACKs update to ensure the file stays sorted but hit some cases with unicode names having different value. 

Thanks.

**Comment by merwok** [comment]
The order is not 100% exact, so I think trying to make an automated order would be hard (e.g. if there are three bits in the name, could be two first names or a two-part last name).

Let me make a PR to add the comment at the end and add it to the backports PRs too (and please review these!)

**Pull request merged**